environment.yml

```diff
@@ -0,0 +1,13 @@
+# Environment for ipyparaview with jupyterlab
+# python 3.7?
+name: ipy_dev
+channels:
+- conda-forge
+dependencies:
+- python=3.7.4
+- jupyter
+- jupyterlab
+- ipython
```

Comment:
We should probably add ipywidgets to this list

Reply:
Added to list and merged your changes.